lib/request.ts

```diff
@@ -16,8 +20,8 @@ export interface ServiceClientRequestOptions extends RequestOptions {
   pathname: string;
   query?: object;
   timing?: boolean;
-  dropRequestAfter?: number;
```

Comment:
Is it possible to map this to a new functionality? I would like to void releasing a breaking change before next major version...